lms/templates/video.html

```diff
@@ -53,9 +53,9 @@ <h4 class="hd hd-5">${_('Video')}</h4>
         % endif
         % if track:
         <div class="wrapper-download-transcripts">
-            <h4 class="hd hd-5" id="video-transcripts">${_('Transcripts')}</h4>
+            <h4 class="hd hd-5" id="video-transcripts_${id}">${_('Transcripts')}</h4>
             % if transcript_download_format:
-            <ul class="list-download-transcripts" aria-describedby="video-transcripts">
+            <ul class="list-download-transcripts" aria-describedby="video-transcripts_${id}">
```

Comment:
I don't think an aria-describedby here is necessary.  Unordered lists don't typically have an accessible name.  I only worry because VO will sometimes add an unintended role to elements like this if there is an aria-* applied to them.